docs/index.rst

```diff
@@ -1,53 +1,211 @@
+           we need a guide, which also includes which services are
+           incompatible with fips   
+
+       * **CIS benchmarking**
+
+         * Center for Internet Security `(CIS) Benchmark`_ tooling is available
+           with ``cis``, up to (and including) 20.04 LTS.
+         * In the 20.04 LTS we introduced the `Ubuntu Security Guide (USG)`_
+           tooling through ``usg``, which replaces ``cis`` in releases after
+           20.04 LTS. 
+
+         * Find out :doc:`how to manage CIS and USG<howtoguides/enable_cis>`
+           with the Pro Client.
+
+       * **Common Criteria**
+
+         * `Common Criteria EAL2 (CC EAL)`_ certification tooling is available
+           through ``cc-eal``. The Ubuntu 18.04 LTS and 16.04 LTS have both
+           been certified.
+
+         * Find out :doc:`how to enable CC EAL<howtoguides/enable_cc>` on
+           your LTS.
+
+       Want more information?
+
+    .. tab-item:: Personal user on LTS
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using Pro on an LTS machine (Desktop or Server) and don't
+         want to upgrade to a newer LTS yet, you can get additional security
+         support through `Expanded Security Maintenance (ESM)`_. We recommend
+         it for everyone using Pro, and it's enabled by default. It includes
+         two services:
+
+         * ``esm-infra`` provides security updates for packages in the Ubuntu
+           'main' repository (packages from Ubuntu) for five additional years
+           beyond the standard five years of security maintenance of 'main'
+           packages.
+
+         * ``esm-apps`` covers packages in the 'universe' repository (which
+           are from Debian and the community) for ten years after the LTS
+           release. 
+
+         * Read more
+           :doc:`about ESM-Apps and ESM-Infra<explanations/about_esm>`.
+
+       * **Livepatch**
+
+         Ubuntu `Livepatch`_ removes the need to immediately reboot your
+         machine after a patch is applied to the kernel, so you can perform
+         device reboots when it's convenient for you. Although it's
+         lightweight, it may not be suitable on a very small system with
+         limited memory. Check our `list of supported kernels`_ if you're
+         unsure.
+
+         * Find out
+           :doc:`how to enable Livepatch<howtoguides/enable_livepatch>`
+
+       Want more information?
+
+    .. tab-item:: Other security services
+
+       * **ROS ESM**
+
+         Our Robot Operating System Expanded Security Maintenance (`ROS ESM`_)
+         service provides security maintenance for ROS LTS releases, starting
+         with ROS Kinetic on Ubuntu 16.04 -- including packages from the Ubuntu
+         'universe' repository.
+
+         * ``ros`` provides all of the same security coverage as you get with
+           ESM, plus more than 600 packages for ROS 1 Kinetic and Melodic, and
+           ROS 2 Foxy. 
```

Comment:
`ros` sits on top of `esm-infra` and `esm-apps` (you're required to enable `esm-infra/apps` to enable `ros`. So it feels a little confusing to say "ros provides all of the same security coverage as you get with ESM", because `ros` isn't the thing providing those updates; esm-infra/apps is. `ros` only provides the updates to "more than 600 packages for ROS..."

Reply:
That distinction is sort of lost in the marketing pages I think, because I didn't pick up on it. Is it worth including (i.e. pointing out that ESM is still required), or should I just remove that entirely and leave only the ros-specific part?

Reply:
I think it's probably better to have it there than not. I've added it now :)